pages/DetailsTrip/[city]/index.js

```diff
@@ -93,60 +94,66 @@ export default function DetailsOfTrip() {
         <StyledH4>Hotels:</StyledH4>
-        <StyledUl>
-          {city.hotels.map((hotel) => (
-            <StyledList key={uuidv4()}>
-              <StyledSpan>{hotel.hotel}</StyledSpan>
-              {hotel.hotelPrice ? (
-                <StyledSpan>{hotel.hotelPrice} Euro</StyledSpan>
-              ) : null}
-            </StyledList>
-          ))}
+        <StyledBlox>
+          <StyledUl>
+            {city.hotels.map((hotel) => (
+              <StyledList key={uuidv4()}>
+                <StyledSpan>{hotel.hotel}</StyledSpan>
+                {hotel.hotelPrice ? (
+                  <StyledSpan>{hotel.hotelPrice} Euro</StyledSpan>
+                ) : null}
+              </StyledList>
+            ))}
 
+            <StyledDiv>
+              <StyledTotalPrice>Total Price:</StyledTotalPrice>
+              <StyledTotalPrice>{totalPriceOfHotels} Euro</StyledTotalPrice>
+            </StyledDiv>
+          </StyledUl>
+        </StyledBlox>
+        <StyledH4>Places to visit:</StyledH4>
+        <StyledBlox>
+          <StyledUl>
+            {city.places.map((place) => (
+              <StyledList key={uuidv4()}>
+                <StyledSpan>{place.place}</StyledSpan>
+                {place.placePrice ? (
+                  <StyledSpan> {place.placePrice} Euro </StyledSpan>
+                ) : null}
+              </StyledList>
+            ))}
+          </StyledUl>
           <StyledDiv>
             <StyledTotalPrice>Total Price:</StyledTotalPrice>
-            <StyledTotalPrice>{totalPriceOfHotels} Euro</StyledTotalPrice>
+            <StyledTotalPrice>{totalPriceOfPlaces} Euro</StyledTotalPrice>
           </StyledDiv>
-        </StyledUl>
-
-        <StyledH4>Places to visit:</StyledH4>
-        <StyledUl>
-          {city.places.map((place) => (
-            <StyledList key={uuidv4()}>
-              <StyledSpan>{place.place}</StyledSpan>
-              {place.placePrice ? (
-                <StyledSpan> {place.placePrice} Euro </StyledSpan>
-              ) : null}
-            </StyledList>
-          ))}
-        </StyledUl>
-        <StyledDiv>
-          <StyledTotalPrice>Total Price:</StyledTotalPrice>
-          <StyledTotalPrice>{totalPriceOfPlaces} Euro</StyledTotalPrice>
-        </StyledDiv>
-
+        </StyledBlox>
         <StyledH4>Food to try:</StyledH4>
-        <StyledUl>
-          {city.food.map((food) => (
-            <StyledList key={uuidv4()}>
-              <StyledSpan>{food.foodName}</StyledSpan>
-              {food.foodPrice ? (
-                <StyledSpan>{food.foodPrice} Euro</StyledSpan>
-              ) : null}
-            </StyledList>
-          ))}
-        </StyledUl>
-        <StyledDiv>
-          <StyledTotalPrice>Total Price:</StyledTotalPrice>
-          <StyledTotalPrice> {totalPriceOfFoods} Euro</StyledTotalPrice>
-        </StyledDiv>
+        <StyledBlox>
+          <StyledUl>
+            {city.food.map((food) => (
+              <StyledList key={uuidv4()}>
```

Comment:
If the food object has an id already on it, then i think it would be best to use that. 

```suggestion
              <StyledList key={food.id}>
```